### ISSUE TITLE
Memory controller cgroup enables

### ISSUE THREAD

**Issue by agherzan** [balena-os/leviathan#99]
Supervisor relies on this cgroup to be able to manage memory for containers so we need a test case which reads /proc/cgroups and makes sure that memory controller is available and enabled. Here is an example on how that file presents this data:

#subsys_name    hierarchy       num_cgroups     enabled
cpuset  7       4       1
cpu     4       4       1
cpuacct 4       4       1
blkio   2       4       1
memory  0       1       1
devices 6       83      1
freezer 5       4       1
net_cls 3       4       1

So basically the last number should be 1 for memory.

Ref.: https://github.com/resin-os/resin-raspberrypi/pull/164
